httpd/handler.go

```diff
@@ -148,7 +116,7 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, u *User) {
 
 	// If authentication is enabled and there are no users yet, make sure
 	// the first statement is creating a new cluster admin.
-	if h.AuthenticationEnabled && h.server.UserCount() == 0 {
+	if h.requireAuthentication && h.server.UserCount() == 0 {
```

Comment:
I think `authenticationEnabled` -- just lower-casing the existing variable name -- is a little better.